examples/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Do you think we need another license here? Also, I thought the examples should be free to use. (Not sure though)

Reply:
I'm not sure whether we should mix licenses within one repository.

Reply:
Let's check later. Better to have one for now